test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -235,38 +235,71 @@ def forward(self, x):
                 x = self.bn(x)
                 return x
 
-        import torch.ao.quantization._pt2e.quantizer.qnnpack_quantizer as qq
-        quantizer = QNNPackQuantizer()
-        quantizer.set_global(qq.get_symmetric_quantization_config(is_per_channel=True, is_qat=True))
-        m = M()
         example_inputs = (torch.randn(1, 3, 5, 5),)
+        self._verify_symmetric_qnnpack_qat_graph(M(), example_inputs, is_per_channel=False, has_relu=False)
+        self._verify_symmetric_qnnpack_qat_graph(M(), example_inputs, is_per_channel=True, has_relu=False)
 
-        # program capture
+    def test_prepare_qat_conv_bn_relu_fusion(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 3)
+                self.bn = torch.nn.BatchNorm2d(3)
+                self.relu = torch.nn.ReLU()
```

Comment:
nit: maybe you can put model definition in helper as well and you can do:
```
self.relu = torch.nn.ReLU() if has_relu else torch.nn.Identity()
```

Reply:
actually if we want to use that function for linear - relu etc. as well it might be OK

Reply:
Yeah actually I'm about to use this helper function for resnet/mobilenet numerics